app/forms/pages/conditions_form.rb

```diff
@@ -40,7 +40,8 @@ def routing_answer_options
   end
 
   def goto_page_options
-    [OpenStruct.new(id: nil, question_text: I18n.t("helpers.label.pages_conditions_form.default_goto_page_id")), form.pages.map { |p| OpenStruct.new(id: p.id, question_text: p.question_text) }, OpenStruct.new(id: "check_your_answers", question_text: I18n.t("page_conditions.check_your_answers"))].flatten
+    page_options = pages_after_current_page(form.pages, page).map { |p| OpenStruct.new(id: p.id, question_text: "#{p.position}. #{p.question_text}") }
```

Comment:
There's an existing method on the page model for getting the question text with the number prepended to it:

```suggestion
    page_options = pages_after_current_page(form.pages, page).map { |p| OpenStruct.new(id: p.id, question_text: p.question_with_number ) }
```